contracts/merger/PegExchanger.sol

```diff
@@ -0,0 +1,60 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "./MergerBase.sol";
+
+/// @title Contract to exchange RGT with TRIBE post-merger
+/// @author elee
+contract PegExchanger is MergerBase {
+    using SafeERC20 for IERC20;
+
+    uint256 public constant MIN_EXPIRY_WINDOW = 180 days;
+
+    uint256 public constant exchangeRate = 26705673430; // 26.7 TRIBE / RGT
```

Comment:
this exchange rate should be scaled by 1e18, instead of 1e9

Reply:
Doesn't matter, its just fixed point precision. We are using 1e9 everywhere here

Reply:
its a custom scalar for the exchange rate, not the tokens ya

tho 18 would be good just for typical convention following